templates/projects/hello-world/files/common/models/message.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "message",
```

Comment:
One more thing: since the Message model is not persisted in any datasource, it should specify `"base": "Model"` so that CRUD methods are not included in the REST API.